src/pages/List/List.js

```diff
@@ -1,7 +1,65 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ItemList from './ItemListComponents/ItemList';
+import SelectBox from './SelectBoxComponents/SelectBox';
+import './List.scss';
 
 const List = () => {
-  return <div>d응ㅇ애</div>;
-};
+  const [ItemInfo, setItemInfo] = useState([]);
+
+  //백엔드 데이터 정보
+  // useEffect(() => {
+  //   fetch('http://10.58.7.79:8000/products/categories/list', {
+  //     method: 'POST',
+  //     body: JSON.stringify({
+  //       type_id: 1,
+  //     }),
+  //   })
+  //     .then(res => res.json())
+  //     .then(data => {
+  //       setItemInfo(data.message);
+  //     });
+  // }, []);
 
+  useEffect(() => {
+    fetch('http://localhost:3000/data/MockData.json')
+      .then(res => res.json())
+      .then(data => setItemInfo(data));
+  }, []);
+
+  return (
+    <div className="list">
+      <div className="pageContainer">
+        <h4>남성니트</h4>
+        <nav className="nextPage">
+          <p className="textArea">
+            총 <b>131개</b>의 상품이 있습니다.
+          </p>
+          <ul>
+            <li className="btnPageL">
+              <img src="/images/listpage/btn_first_dim.gif" alt="button" />
+            </li>
+            <li className="btnPageL">
+              <img src="/images/listpage/btn_prev_dim.gif" alt="button" />
+            </li>
+            <li>
+              <strong>1</strong>
+            </li>
+            <li className="btnPageR">
+              <img src="/images/listpage/btn_next_dim.gif" alt="button" />
+            </li>
+            <li className="btnPageR">
+              <img src="/images/listpage/btn_last_dim.gif" alt="button" />
+            </li>
+          </ul>
+        </nav>
+        <SelectBox />
+        <ul className="itemBox">
+          {ItemInfo.map(item => (
+            <ItemList key={item.id} {...item} />
+          ))}
```

Comment:
이름이 좀 더 명확하면 좋을 것 같네요!

Info등의 접미사는 안붙이는게 좋습니다!
Item vs ItmeInfo 로 생각했을 때 어떤 부분이 차이가 있는지 Info라는 단어들로는 명확히 알기 힘들겠죠??

상품에 대한 정보라면 그리고 Item이라는 단어도 좀 불분명한것 같아서 상품에 대한 정보라면 products정도로 표현하는게 좋겠네요!

```suggestion
          {products.map(product => (
            <ItemList key={product.id} {...product} />
          ))}
```

그러면 위와 같이 좀 더 알아보기 쉬운 형태가 되겠네요!!
ItemList라는 컴포넌트 명도 실질적으로 리스트가 아닌 단일 아이템을 표현하는 컴포넌트 같으니까 수정해주는게 좋겠네요!

Reply:
네 감사합니다! 수정하였습니다~